include/tube_manager.h

```diff
@@ -13,6 +13,7 @@
 #include "spud.h"
 #include "ls_error.h"
 #include "ls_event.h"
+#include "ls_htable.h"
```

Comment:
I've been wanting to move ls_htable.h out of the externally-visible includes.  Can you copy the walker function into tube_manager.h, rename it, then cast it in the implementation, please?